### PR TITLE
Add examples in evaluation docstring

### PR DESCRIPTION
**Pull request by zhu0619**
## Changelogs

- Added example in `benchmark.evaluation` docstrings to clarify the evaluation inputs.

### CODE REVIEW

**Review by Andrewq11 — COMMENTED** [review]
Thanks @zhu0619! Left some comments about the docstring and test.

File: polaris/benchmark/_base.py [hunk 1]
```diff
@@ -406,6 +406,19 @@ def evaluate(
 
         Returns:
             A `BenchmarkResults` object. This object can be directly submitted to the Polaris Hub.
+
+
+        Examples:
+            1. For regression benchmarks:
+                predictions = model.predict(molecules)
+                benchmark.evaluate(y_pred=prediction_labels)
```
[the remark this answers is not in the document]
Comment (zhu0619) [reply polaris/benchmark/_base.py]: I updated the naming for consistency.

File: polaris/benchmark/_base.py [hunk 2]
```diff
@@ -406,6 +406,19 @@ def evaluate(
...
+            2. For classification benchmarks:
+                - If `roc_auc` and `pr_auc` in the metric list, both class probabilities and label predictions are required:
+                    pred_probs = model.predict_proba(molecules) # predict probablities
+                    pred_labels = model.predict_lables(molecules) # predict class labels
+                    benchmark.evaluate(y_pred=prediction_labels, y_prob=pred_labels)
+                - Otherwise:
+                    benchmark.evaluate(y_pred=prediction_labels)
```
[the remark this answers is not in the document]
Comment (zhu0619) [reply polaris/benchmark/_base.py]: I updated the naming for consistency and clarification.

File: tests/test_evaluate.py [hunk 3]
```diff
@@ -169,7 +169,7 @@ def test_metric_y_types(
 
     # If y_type != "y_pred" and y_prob is None, an error is thrown.
     with pytest.raises(ValueError, match="Metric.roc_auc requires `y_prob` input"):
-        test_single_task_benchmark_clf.metrics = [Metric.roc_auc]
+        test_single_task_benchmark_clf.metrics = [Metric.roc_auc, Metric.pr_auc]
```
[the remark this answers is not in the document]
Comment (zhu0619) [reply tests/test_evaluate.py]: You are right. I added another test for `pr_auc`.

File: polaris/benchmark/_base.py [hunk 4]
```diff
@@ -406,6 +406,19 @@ def evaluate(
 
         Returns:
             A `BenchmarkResults` object. This object can be directly submitted to the Polaris Hub.
+
+
+        Examples:
+            1. For regression benchmarks:
+                pred_scores = your_model.predict_score(molecules) # predict continous score values
```
Comment (Andrewq11) [inline polaris/benchmark/_base.py:413]:
```suggestion
                pred_scores = your_model.predict_score(molecules) # predict continuous score values
```

**Review by Andrewq11 — APPROVED** [review]

**Pull request merged**